site/content/internal/rd-teams.md

```diff
@@ -103,3 +103,6 @@ weight: 110
 * Jelena Gilliam - QA
 * Ogi Marusic - QA
 * Steve Mudie - QA
+* Justine Geffen - Technical Writer
+* Adam Clarkson - PM
```

Comment:
Does it make sense to list the PM with their associated team(s)?

Reply:
They're not yet mapped to a team, from PR description: `Adam/Eric: For R&D list, I added you under "Other" while we figure out how the new product and engineer structure work together.`

Reply:
Hah, teaches me to read the whole description next time -- thanks for the gentle reminder :)